test/m3u8.test.js

```diff
@@ -511,30 +511,35 @@ QUnit.test('parses minimal EXT-X-PROGRAM-DATE-TIME tags', function() {
   QUnit.strictEqual(element.tagType, 'date-time', 'the tag type is date-time');
   QUnit.ok(!('dateTimeString' in element), 'no dateTime is present');
 });
-QUnit.test('parses EXT-X-PROGRAM-DATE-TIME tags with valid date-time formats', function() {
-  let manifest = '#EXT-X-PROGRAM-DATE-TIME:2016-06-22T09:20:16.166-04:00\n';
-  let element;
-
-  this.parseStream.on('data', function(elem) {
-    element = elem;
+QUnit.test('parses EXT-X-PROGRAM-DATE-TIME tags with valid date-time formats',
+  function() {
+    let manifest = '#EXT-X-PROGRAM-DATE-TIME:2016-06-22T09:20:16.166-04:00\n';
+    let element;
+
+    this.parseStream.on('data', function(elem) {
+      element = elem;
+    });
+    this.lineStream.push(manifest);
+
+    QUnit.ok(element, 'an event was triggered');
+    QUnit.strictEqual(element.type, 'tag', 'the line type is tag');
+    QUnit.strictEqual(element.tagType, 'date-time', 'the tag type is date-time');
```

Comment:
I believe the `tagType` should be `program-date-time` to be consistent with other tags.